include/cinder/ConcurrentCircularBuffer.h

```diff
@@ -107,7 +107,13 @@ class ConcurrentCircularBuffer : private Noncopyable {
 	}
 	
 	//! Returns the number of items the buffer can hold
-	size_t size() const { return (size_t)mContainer.capacity(); }
+	size_t getCapacity() const { return (size_t)mContainer.capacity(); }
+	
+	//! Returns the number of items the buffer is currently holding
+	size_t getSize() {
```

Comment:
Hah, I realize in the linked issue I suggested implementing `getSize()` with a mutex, but then now that I'm thinking about it doesn't seem necessary.